zookeeper-server/src/main/resources/lib/jetty-http-9.4.35.v20201120.LICENSE.txt

```diff
@@ -1,8 +1,7 @@
 This program and the accompanying materials are made available under the
-terms of the Eclipse Public License 1.0 which is available at
-https://www.eclipse.org/org/documents/epl-1.0/EPL-1.0.txt
-or the Apache Software License 2.0 which is available at
-https://www.apache.org/licenses/LICENSE-2.0
+terms of the Eclipse Public License 2.0 which is available at
```

Comment:
Please remove this license file as 3.5 no longer uses jetty-client. Thank you!

Reply:
I'm seeing my comment on jetty-http on the main page, but on "files" page I commented on jetty-client. Strange. But it's the jetty-client that we don't need.

Reply:
> Please remove this license file as 3.5 no longer uses jetty-client. Thank you!
>
> I'm seeing my comment on jetty-http on the main page, but on "files" page I commented on jetty-client. Strange. But it's the jetty-client that we don't need.

I had already removed `jetty-client` on this branch:

```
$ ls zookeeper-server/src/main/resources/lib/jetty-*
zookeeper-server/src/main/resources/lib/jetty-http-9.4.35.v20201120.LICENSE.txt
zookeeper-server/src/main/resources/lib/jetty-io-9.4.35.v20201120.LICENSE.txt
zookeeper-server/src/main/resources/lib/jetty-security-9.4.35.v20201120.LICENSE.txt
zookeeper-server/src/main/resources/lib/jetty-server-9.4.35.v20201120.LICENSE.txt
zookeeper-server/src/main/resources/lib/jetty-servlet-9.4.35.v20201120.LICENSE.txt
zookeeper-server/src/main/resources/lib/jetty-util-9.4.35.v20201120.LICENSE.txt
zookeeper-server/src/main/resources/lib/jetty-util-ajax-9.4.35.v20201120.LICENSE.txt
```

The reason you are observing a mix-up, and your comment landed in a strange place, is because Git thinks I have renamed `jetty-client-9.4.34` to `jetty-http-9.4.35`, etc., and that I have deleted `jetty-server-9.4.34`!  (Similar contents and different file names throw its rename detector off.)

The result is correct.  The Git commit is correct.  It's only the diff which *appears* wrong (it is actually correct, but overly convoluted).

So unless I'm missing something else, I think we're good.